### DISCUSSION TITLE
Get album by barcode id

### DISCUSSION THREAD

**Discussion by filippomenchini** [tidal-music/discussions#21] [Q&A] [answered]
Hello everyone!

While reading the docs I noticed that the "Get album by barcode id" returns a list of albums. I managed to use this endpoint without any problem, but shouldn't it return only one album? I'm curious about this, I thought that barcode ids were unique to each album released, am I wrong?

**Comment by aliaksei-taliuk** [comment, marked as answer]
Hi @filippomenchini ! 

'barcode id' (which is also called UPC for albums and ISRC for tracks) doesn't uniquely identify the entity. The same item can be re-released many times with different providers. E.g. XXX 2014 Remastered or something like that. 

Thus, it has to be an array.

Please let us know if you still have any questions.

Kind regards,
Alexey

**Reply by filippomenchini** [reply]
Got it.

Thanks for the info!
